### PR TITLE
requirement layout and versions changes

### PR DESCRIPTION
**Pull request by zeograd**
monolithic requirements.txt in a documentation directory
doesn't follow the "one true way" django recommendations
(cf Kaplan-moss' talk at OSCON 2011).
This pull request propose to split and move it, as well
as update minimally the versions of requirements to be
installable as of today (2016-02-05).

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+# PIL==1.1.6
+Pillow==1.7.8
 South==0.7.3
 Whoosh==2.3
 django-extensions==0.5
 django-haystack==1.2.6
-#django-tagging==0.3.1
--e git+http://git.gitorious.org/demovibes/django-tagging-fork.git#egg=django_tagging_fork
-python-memcached==1.47
+-e git+https://gitlab.com/zeograd/django-tagging-fork.git#egg=django_tagging_fork
```
Comment (arabek) [inline requirements/base.txt]: Why this change when nothing was pushed to the upstream (fork) nor the fork of a fork? :)

File: requirements/base.txt [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+
+# python-memcached==1.47
+
 python-openid==2.2.5
 wsgiref==0.1.2
 flup==1.0.3.dev-20110405
-MySQL-python==1.2.3
+# MySQL-python==1.2.3
```
Comment (arabek) [inline requirements/base.txt]: Why the comment here?

Comment (zeograd) [inline requirements/base.txt]: database driver should, imho, be matching the platform specific settings.py.
Since the committed settings.py is configured for using sqlite, I think that the base.txt requirement shouldn't enforce installing a mysql driver.
However, as the settings.py should be broken down into a settings/prod.py which has mysql configured as database, the requirements/prod.txt should contain the mysql driver.
However, feel free to let the mysql driver in base.txt, it's a rather light requirement and not conflicting with anything.

File: requirements/base.txt (hunk 1 above)
Comment (zeograd) [inline requirements/base.txt]: the repository on gitorious.org isn't accessible anymore as far as I can tell. I had installed demovibes on my laptop several days earlier and at that time I had managed to clone the django-tagging-fork.
My proposal to switch to the repository on gitlab is only for the sake of repository reachability.
The final target is to use the upstream project, but in the meantime, demovibes remains installable.

Comment (arabek) [inline requirements/base.txt]: Seems ok to me. Please revert the change.

[ arab@navarro: ~/src ]$ git clone http://git.gitorious.org/demovibes/django-tagging-fork.git
Cloning into 'django-tagging-fork'...
remote: Counting objects: 691, done.
remote: Compressing objects: 100% (143/143), done.
remote: Total 691 (delta 529), reused 677 (delta 521)
Receiving objects: 100% (691/691), 131.11 KiB | 0 bytes/s, done.
Resolving deltas: 100% (529/529), done.
Checking connectivity... done.
[ arab@navarro: ~/src ]$

File: requirements/base.txt (hunk 2 above)
Comment (arabek) [inline requirements/base.txt]: :+1:

Comment (zeograd) [inline requirements/base.txt]: I propose to make this even clearer by adding a requirements/prod.txt in which we complement the base requirements with the mysql driver.
Should I update the pull request with it ?

Comment (wojciecharabczykpayleven) [inline requirements/base.txt]: If that's not a problem for you? Please do :)

Comment (arabek) [inline requirements/base.txt]: If that's not a problem for you, please do :)

Comment (zeograd) [inline requirements/base.txt]: No problem, it's added :)
